src/test/java/com/wikia/webdriver/elements/mercury/pages/InfoboxBuilderPage.java

```diff
@@ -1,16 +1,16 @@
-package com.wikia.webdriver.elements.oasis.pages;
+package com.wikia.webdriver.elements.mercury.pages;
 
+import com.wikia.webdriver.common.contentpatterns.URLsContent;
 import com.wikia.webdriver.common.core.Assertion;
```

Comment:
Few unused fields in this class: 
+ helpDialog
+ successIcon
+ sectionHeadersChevron